<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>490</width>
    <height>330</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>163</red>
        <green>244</green>
        <blue>244</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>163</red>
        <green>244</green>
        <blue>244</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>163</red>
        <green>244</green>
        <blue>244</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>170</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Qt购物系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>491</width>
      <height>281</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Manager">
     <widget class="QPushButton" name="loginButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>用户注册</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="inquireButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>条件查询</string>
      </property>
     </widget>
     <widget class="QPushButton" name="goodsMngButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>商品管理</string>
      </property>
     </widget>
     <widget class="QPushButton" name="purchaseButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>购买商品</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ordersButton">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>订单总览</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_4" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_6" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>241</width>
        <height>191</height>
       </rect>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_7" native="true">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>210</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_8" native="true">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>210</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="corpMngButton">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>快递公司管理</string>
      </property>
     </widget>
     <widget class="QPushButton" name="StateButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>210</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>修改订单状态</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Statistics">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>271</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;title&gt;新建网页 1&lt;/title&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;关于Qt购物系统&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt;Created by Stille on 15/9/18.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:11pt; font-style:italic;&quot;&gt;Copyright (c) 2015年 Stille. All rights reserved.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;The program is provided as A SHOPPING SYSTEM FOR MANAGERS, INCLUDING NECESSARY FUNCTIONS YOU NEED.If you find any ERRORs or INCONVENIENCE, &lt;/span&gt;&lt;span style=&quot; font-size:12pt; text-decoration: underline;&quot;&gt;please FEEDBACK to us !&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Contact Us</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>111</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>旧密码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>新密码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>确认新密码：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>200</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>确认修改</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>重置</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_9" native="true">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>201</width>
        <height>80</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>201</width>
         <height>81</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>总管理</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="separator"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>修改密码</string>
    </property>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>购物管理</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>销售统计</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>更改</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>错误反馈</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
